pkgs/applications/version-management/codeberg-cli/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, rustPlatform
+, fetchFromGitea
+, pkg-config
+, openssl
+, stdenv
+, Security
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "codeberg-cli";
+  version = "v0.3.5";
```

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming
```suggestion
  version = "0.3.5";
```